test/packages/aws/data_stream/cloudtrail/_dev/test/pipeline/test-add-user-to-group-json.log-expected.json

```diff
@@ -19,7 +19,7 @@
                 "ip": "127.0.0.1"
             },
             "event": {
-                "ingested": "2021-03-18T12:21:57.668559300Z",
+                "ingested": "2021-04-19T09:58:42.209230300Z",
```

Comment:
Do you know why this is needed?

Reply:
I regenerated test results as it's simple for us to operate, but it still failed somewhere. Investigating.